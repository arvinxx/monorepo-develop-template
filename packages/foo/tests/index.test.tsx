import { render } from '@testing-library/react';

import Foo from '@arvinxu/foo';

test('Foo', () => {
  const { container } = render(<Foo />);

  expect(container).toMatchSnapshot();
});
