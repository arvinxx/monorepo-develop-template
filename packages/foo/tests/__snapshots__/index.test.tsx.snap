// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Foo 1`] = `
<div>
  <div>
    Hello! 
  </div>
</div>
`;
